### PR TITLE
Several meta data updates

### PR DESCRIPTION
**Pull request by sevyakenza**

### CODE REVIEW

File: types/elsys/elt-2/meta.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ELT-2",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/elt-2/meta.json:3]: @sevyakenza This is the version of our script, not the version of the sensor. Should be 1.0.0

File: types/browan/TBClimate/meta.json [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
   "author": "Akenza AG",
   "firmwareVersion": "V1.0.0",
   "loraDeviceClass": "A",
   "availableSensors": [
     "Temperature",
-    "Humidity",
-    "CO2",
-    "VOC",
-    "Reed"
+    "Humidity"
```
Comment (J2thatsme) [inline types/browan/TBClimate/meta.json:12]: At least Co2 and VOC should be included https://core.akenza.io/device/0297eaf160518c02?environment=2997cd32857f846d

File: types/elsys/elt-lite/meta.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ELT Lite",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/elt-lite/meta.json:3]: Version 1.0.0

File: types/elsys/ems-desk/meta.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "EMS Desk",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ems-desk/meta.json:3]: Version 1.0.0

File: types/elsys/ems-door/meta.json [hunk 5]
```diff
@@ -1,15 +1,13 @@
 {
   "name": "EMS Door",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ems-door/meta.json:3]: Version 1.0.0

File: types/elsys/ems-lite/meta.json [hunk 6]
```diff
@@ -1,23 +1,21 @@
 {
   "name": "EMS Lite",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ems-lite/meta.json:3]: Version 1.0.0

File: types/elsys/ems/meta.json [hunk 7]
```diff
@@ -1,23 +1,23 @@
 {
   "name": "EMS",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ems/meta.json:3]: Version 1.0.0

File: types/elsys/ers-co2-lite/meta.json [hunk 8]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ERS CO2 Lite",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers-co2-lite/meta.json:3]: Version 1.0.0

File: types/elsys/ers-co2/meta.json [hunk 9]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ERS CO2",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers-co2/meta.json:3]: Version 1.0.0

File: types/elsys/ers-desk/meta.json [hunk 10]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ERS Desk",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers-desk/meta.json:3]: Version 1.0.0

File: types/elsys/ers-eye/meta.json [hunk 11]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ERS Eye",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers-eye/meta.json:3]: Version 1.0.0

File: types/elsys/ers-lite/meta.json [hunk 12]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ERS Lite",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers-lite/meta.json:3]: Version 1.0.0

File: types/elsys/ers-sound/meta.json [hunk 13]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "ERS Sound",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers-sound/meta.json:3]: Version 1.0.0

File: types/elsys/ers/meta.json [hunk 14]
```diff
@@ -1,22 +1,22 @@
 {
   "name": "ERS",
-  "version": "1.0.0",
+  "version": "2.3.5",
```
Comment (J2thatsme) [inline types/elsys/ers/meta.json:3]: Version 1.0.0

**Review by J2thatsme — CHANGES_REQUESTED** [review]
Some small changes & one question

**Comment by J2thatsme** [comment]
FYI @mariusgiger here a list of the changed names/ids
`O2, VOC luminaire
Temperature & Humidity Sensor, TBHH100, Tabs
Door & Window Sensor, TBDW100 Tabs
Sound Level Sensor, TBSL100, Tabs
Water Leak Sensor, TBWL100, Tabs
LPN Multi-Sensor 3
ULTRAHEAT T550
FMLR IoT Button V2.5.0
MLR IoT Button V2.0.0
R312A - Wireless Emergency Button
Door R311A-Wireless Door/Window Sensor (LoRaWAN)
R711-Wireless Temperature and Humidity Sensor
Buttonboard (LoRaWAN)
Oxobutton Q, (LoRaWAN)
PlacePod (Surface & In-Ground Mount)
All Strips (LoRaWAN)
Enlink DP
Enlink T`

File: types/browan/TBClimate/meta.json [hunk 15]
```diff
@@ -1,23 +1,20 @@
 {
-  "name": "Temperature & Humidity Sensor, TBHH100, Tabs",
+  "name": "Tabs Temperature & Humidity Sensor, TBHH100",
```
Comment (mariusgiger) [inline types/browan/TBClimate/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/browan/TBDW100/meta.json [hunk 16]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Door & Window Sensor, TBDW100 Tabs",
+  "name": "Tabs Door & Window Sensor, TBDW100",
```
Comment (mariusgiger) [inline types/browan/TBDW100/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/browan/TBSound/meta.json [hunk 17]
```diff
@@ -1,21 +1,20 @@
 {
-  "name": "Sound Level Sensor, TBSL100, Tabs",
+  "name": "Tabs Sound Level Sensor, TBSL100",
```
Comment (mariusgiger) [inline types/browan/TBSound/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/browan/TBWaterleak/meta.json [hunk 18]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Water Leak Sensor, TBWL100, Tabs",
+  "name": "Tabs Water Leak Sensor, TBWL100",
```
Comment (mariusgiger) [inline types/browan/TBWaterleak/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/landisgyr/T550/meta.json [hunk 19]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "ULTRAHEAT T550",
+  "name": "WZU-LR",
```
Comment (mariusgiger) [inline types/landisgyr/T550/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/miromico/REV2_V5/meta.json [hunk 20]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "FMLR IoT Button V2.5.0",
+  "name": "miro Click V2.5.0",
```
Comment (mariusgiger) [inline types/miromico/REV2_V5/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/miromico/sod/meta.json [hunk 21]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "FMLR IoT Button V2.0.0",
+  "name": "miro Click V2.0.0",
```
Comment (mariusgiger) [inline types/miromico/sod/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/netvox/button/meta.json [hunk 22]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "R312A - Wireless Emergency Button",
+  "name": "R312A",
```
Comment (mariusgiger) [inline types/netvox/button/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/netvox/door/meta.json [hunk 23]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Door R311A-Wireless Door/Window Sensor (LoRaWAN)",
+  "name": "R311A",
```
Comment (mariusgiger) [inline types/netvox/door/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/netvox/r711/meta.json [hunk 24]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "R711-Wireless Temperature and Humidity Sensor",
+  "name": "R711",
```
Comment (mariusgiger) [inline types/netvox/r711/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/oxon/buttonboard/meta.json [hunk 25]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Buttonboard (LoRaWAN)",
+  "name": "Buttonboard",
```
Comment (mariusgiger) [inline types/oxon/buttonboard/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/oxon/oxobutton/meta.json [hunk 26]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Oxobutton Q, (LoRaWAN)",
+  "name": "Oxobutton Q",
```
Comment (mariusgiger) [inline types/oxon/oxobutton/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/pnicorp/placePod/meta.json [hunk 27]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "PlacePod (Surface & In-Ground Mount)",
+  "name": "PlacePod",
```
Comment (mariusgiger) [inline types/pnicorp/placePod/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/sensative/strip/meta.json [hunk 28]
```diff
@@ -1,19 +1,15 @@
 {
-  "name": "All Strips (LoRaWAN)",
+  "name": "Strips Multi-sensor +Guard",
```
Comment (mariusgiger) [inline types/sensative/strip/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/synetica/enlinkStatusDP/meta.json [hunk 29]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Enlink DP",
+  "name": "enLink Status-DP",
```
Comment (mariusgiger) [inline types/synetica/enlinkStatusDP/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/synetica/enlinkStatusT/meta.json [hunk 30]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Enlink T",
+  "name": "enLink Status-T",
```
Comment (mariusgiger) [inline types/synetica/enlinkStatusT/meta.json:2]: changes in name will need to be done manually, therefore coordination when merging this is needed

File: types/browan/TBClimate/meta.json (hunk 2 above)
Comment (sevyakenza) [reply types/browan/TBClimate/meta.json:12]: @J2thatsme this sensor was probably mixed with the IAQ Healthy Home sensor from browan which provides CO2 and VOC. (sensor which you have linked)
The TBHH100 which is the sensor in this case does not provide any CO2 or VOC value.

**Review by mariusgiger — APPROVED** [review]

**Review by mariusgiger — DISMISSED** [review]

**Pull request merged**